src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+/**
+ * Calculates the rental cost based on the number of days rented,
+ * applying different discounts based on the duration.
+ *
+ * @param {number} days - The number of days the item is rented for.
+ * @return {number} The total rental cost.
+ */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+
+  if (days < 3) {
+    return days * dailyRate;
+  } else if (days >= 3 && days <= 6) {
```

Comment:
Prefer using 'if' with 'return' over 'if...else' to simplify the flow of the function. After a return statement, the 'else' is unnecessary since the code following it won't be executed if the condition is met.